src/content/docs/browser/browser-integrations/mexn-integration.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: MExN integration
+tags:
+    - New Relic integrations
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
+---
+
+
+import browserMeanMernIntegtrationDashboard from 'images/browser_screenshot-full_mean-mern-integtration-dashboard.png'
+
+MEAN and MERN stacks are open-source full stack software development kits. Both kits use a single codebase to develop web applications. New Relic has combined both stacks into a single quickstart. Our MExN integration makes use of the Browser agent, Node,js agent, and MongoDB agent to provide you with a pre-built dashboard with your most important metrics (like response time, cpu utilization, traffic and login frequencies).
```

Comment:
have we confirmed that it's definitely a 'mongodb agent'? I've never heard of another agent besides the main ones, so I'd like to confirm from someone pretty high up that it should be called an agent and not just an integration or something else.

Reply:
CPU capitalization

Reply:
done!